dockerfiles/init/manifests/codenvy.pp

```diff
@@ -303,6 +303,12 @@
   $db_schema_flyway_scripts_locations=getValue("DB_SCHEMA_FLYWAY_SCRIPTS_LOCATIONS","classpath:che-schema,classpath:codenvy-schema")
 
 ###############################
+# Rsync backup/restore configuration
+  $rsync_restore_bwlimit=getValue("RSYNC_RESTORE_BWLIMIT","7000")
```

Comment:
what do you mean "change" ? if you want to rename it let's rename them now :)
in general im +1 to not add this  to codenvy.env  for now but we need explain somehow how those props can be used.